### PR TITLE
fix(ui): Avoid submitting form if player is inside form

### PR DESCRIPTION
**Pull request by WINOFFRG**
fixes https://github.com/shaka-project/shaka-player/issues/4861

`button` element by default has `type` set to `submit` which can interfere with forms.

### CODE REVIEW

File: lib/util/dom_utils.js [hunk 1]
```diff
@@ -89,4 +91,3 @@ shaka.util.Dom = class {
     }
   }
 };
-
```
Comment (avelad) [inline lib/util/dom_utils.js:92]: Revert this change

**Review by avelad — CHANGES_REQUESTED** [review]

**Review by avelad — APPROVED** [review]

**Review by joeyparrish — APPROVED** [review]
